### DISCUSSION TITLE
Android 15 compatibility

### DISCUSSION THREAD

**Discussion by andrea-musci** [RikkaApps/Shizuku#617] [Q&A]
Hello,
is there any info on Android 15 compatibility?
I'm about to receive the update but holding on 'cause don't want to lose Shizuku.

Thx

**Comment by Magneticdud** [comment]
on my phone with android 15 it seems to lose the pairing after a while, wireless debugging completely stops

**Comment by effleurager** [comment]
Been using Android 15 on my Pixel without any issues

**Reply by Magneticdud** [reply]
It looks like if I disable USB debugging, then wireless debugging will turn off after a few minutes of inactivity , disabling shizuku

**Comment by dashingoo** [comment]
I use it on a Realme GT 6 Neo with Android 15, can't pair it. Waiting for the updating.

**Reply by dashingoo** [reply]
Just now I tried again, succeeded ! maybe USB debugging is open.
